iocore/net/LuaSNIConfig.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+/* 
+ * File:   LuaSNIConfig.h
+ * Author: persia
+ *
+ * Created on September 21, 2017, 4:25 PM
+ */
+
+#ifndef LUASNICONFIG_H
+#define LUASNICONFIG_H
+
+#include "tsconfig/TsConfigLua.h"
+#include "tsconfig/Errata.h"
+using namespace ts;
```

Comment:
No namespace using in a header file. You can use a specific element, e.g.
```
using ts::Errata;
```
but no global import. Too easy to break things.